### PR TITLE
(Publish February 22nd) docs: Browser Agent Release 1226

### PR DESCRIPTION
**Pull request by patrickhousley**
Release notes for Browser Agent release 1226

### CODE REVIEW

**Review by rhetoric101 — APPROVED** [review]
@patrickhousley thanks for adding these release changes!

**Comment by patrickhousley** [comment]
Please wait to publish this. We are still waiting on final approvals to push this change out.

File: src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1226.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## New Features
+
+### Enable back/forward cache
+Updating the agent default configuration to enable the back/forward cache feature previously released in version 1222 by default.
+
+## Bug Fixes
+
+### Revert xhr deny list timeslice metrics
+Customers were losing visibility into all calls on the page when denying timeslice metrics based on the deny list.  This change reverts to the behaviour seen in all previous versions of the Browser Agent.
```
Comment (a-sassman) [inline src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1226.mdx:15]:
```suggestion
Customers were losing visibility into all calls on the page when denying timeslice metrics based on the deny list. This change reverts to the behavior seen in all previous versions of the browser agent.
```

**Review by a-sassman — APPROVED** [review]

**Comment by patrickhousley** [comment]
I added a comment last night that this should not be merged

**Comment by a-sassman** [comment]
@patrickhousley Oh, apologies! I'll revert it and update the PR title.

**Comment by a-sassman** [comment]
@patrickhousley 
I needed to create a new PR, so please make any future updates/comments to this one: https://github.com/newrelic/docs-website/pull/11692 . Again so sorry about that!

**Comment by patrickhousley** [comment]
@ally-sassman NP thanks for rolling that back so quickly and getting it re-staged. Much appreciated!

**Pull request merged**